contracts/MemberToken.sol

```diff
@@ -154,6 +154,8 @@ contract MemberToken is ERC1155Supply, Ownable {
         uint256[] memory amounts,
         bytes memory data
     ) internal override {
+        super._beforeTokenTransfer(operator, from, to, ids, amounts, data);
```

Comment:
we can't redeploy the MemberToken contract

Reply:
hmm okay. this should have been part of the original contract because we can't fetch the total supply from the contract FYI. Too late now I suppose

Reply:
it is, but we overrode the _beforeTokenTransfer and didn't call super.